### PR TITLE
Moved LD_LIBRARY_PATH to java.library.path to address part of wpilibs…

### PR DESCRIPTION
**Pull request by 333fred**
…uite/java-installer#18. Also removed the GenICAM path, as it's no longer necessary. @Kevin-OConnor @PeterJohnson. /cc @sciencewhiz.

### CODE REVIEW

**Comment by sciencewhiz** [comment]
I get farther, but not all the way running.

`Exception in thread "main" java.lang.UnsatisfiedLinkError: /usr/local/frc/lib/li           bwpilibJavaJNI.so: libMathParser_gcc-4.4-arm_v3_0_NI.so: cannot open shared obje           ct file: No such file or directory
        at java.lang.ClassLoader$NativeLibrary.load(Native Method)
        at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1929)
        at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1847)
        at java.lang.Runtime.loadLibrary0(Runtime.java:870)
        at java.lang.System.loadLibrary(System.java:1119)
        at edu.wpi.first.wpilibj.hal.JNIWrapper.<clinit>(JNIWrapper.java:22)
        at edu.wpi.first.wpilibj.RobotBase.initializeHardwareConfiguration(Robot           Base.java:165)
        at edu.wpi.first.wpilibj.RobotBase.main(RobotBase.java:187)
`

libMathParser_gcc-4.4-arm_v3_0_NI.so is in the GenICAM path, so I think it's still necessary.

**Comment by ThadHouse** [comment]
Note this PR should also include a requirement to only use image V8, since it will break with old images.

**Comment by PeterJohnson** [comment]
@sciencewhiz what image version?  v6 or v8?

**Comment by 333fred** [comment]
Why would it break old images, since we apparently still need the GenICAM stuff?

**Comment by ThadHouse** [comment]
Well if it did work without the genIcam stuff, that would have broken old images. But I feel like Joe is potentially on v6 or v7. Id wait for a response from him before adding that stuff back in.

**Comment by sciencewhiz** [comment]
I'm on 7, updating to 8 now. What changed?

**Comment by 333fred** [comment]
/etc/ld.so.conf.d/vision_libs.conf

**Comment by ThadHouse** [comment]
@sciencewhiz Sometime between v6 and v8 they fixed vision_libs. So to test, don't redownload this PR as fred just added a patch to add that back in.

**Comment by sciencewhiz** [comment]
works for me with v8

File: edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/robotCommand [hunk 1]
```diff
@@ -1,2 +1 @@
-env LD_LIBRARY_PATH=/opt/GenICam_v3_0_NI/bin/Linux32_ARM/:/usr/local/frc/lib /usr/local/frc/JRE/bin/java -jar /home/lvuser/FRCUserProgram.jar
-
+/usr/local/frc/JRE/bin/java -Djava.library.path=/usr/local/frc/lib/ -jar /home/lvuser/FRCUserProgram.jar
```
Comment (PeterJohnson) [inline edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/robotCommand:1]: We should be consistent on trailing slash or no trailing slash.

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**